e2e/api/steps_LRspec.js

```diff
@@ -205,6 +205,61 @@ module.exports = {
     deleteCaseFields('applicantSolicitor1CheckEmail');
   },
 
+
+  createClaimSpecFlightDelay: async (user, scenario = 'ONE_V_ONE_FLIGHT_DELAY') => {
+    const pbaV3 = await checkToggleEnabled(PBAv3);
+    eventName = 'CREATE_CLAIM_SPEC';
+    caseId = null;
+    caseData = {};
+
+    let createClaimData  = {};
+
+    createClaimData = data.CREATE_CLAIM(scenario, pbaV3);
+    //==============================================================
+
+    await apiRequest.setupTokens(user);
+    await apiRequest.startEvent(eventName);
+    for (let pageId of Object.keys(createClaimData.userInput)) {
+      await assertValidData(createClaimData, pageId);
+    }
+
+    await assertSubmittedEvent('PENDING_CASE_ISSUED');
+
+    await assignCaseRoleToUser(caseId, 'RESPONDENTSOLICITORONE', config.defendantSolicitorUser);
+
+    if (scenario === 'ONE_V_TWO_SAME_SOL' && createClaimData.userInput.SameLegalRepresentative
```

Comment:
Do we need these other scenarios?

Reply:
Scenarios deleted. I think that they are not necessary.